cmd/util/util.go

```diff
@@ -0,0 +1,39 @@
+package util
+
+import (
+  "fmt"
+  "os"
+  "os/exec"
+  "strings"
+
+  "github.com/fatih/color"
+)
+
+func Contains(list []string, item string) bool {
```

Comment:
Maybe call this method `ContainsString`?

A shame btw that one has to write such methods oneself...